src/posts/context_processors.py

```diff
@@ -9,7 +9,7 @@ def categories(request):
 
 
 def notifications(request):
-    if request.user.is_authenticated:
+    if request.user.is_authenticated and hasattr(request.user, 'userprofile'):
```

Comment:
Maybe add a comment why we added this change. That this is loaded before the `UserProfile` is created and if breaks the signup flow.

Reply:
Done